.gitignore

```
@@ -125,3 +125,6 @@ integration-tests/**/.lock
 
 # Rollup
 .rollup.cache/
+
+# Generated docs
+**/docs/**
```

I would like this to be a little more specific and it might as well be moved up into the "# Build artifacts" section too.

```suggestion
/packages/realm/docs/
```